V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
-
-#### DelegationRequest
-- `public DelegationRequest<T> addParameters(@NonNull Map<String, Object> parameters)`. Use `public DelegationRequest<T> addParameters(@NonNull Map<String, String> parameters)` instead.
-- `public DelegationRequest<T> addParameter(@NonNull String name, @NonNull String value)`. Use `public DelegationRequest<T> addParameter(@NonNull String name, @NonNull String value)` instead.
-
-#### ProfileRequest
-- `public ProfileRequest addParameters(@NonNull Map<String, Object> parameters)`. Use `public ProfileRequest addParameters(@NonNull Map<String, String> parameters)` instead.
-- `public ProfileRequest addParameter(@NonNull String name, @NonNull String value)`. Use `public ProfileRequest addParameter(@NonNull String name, @NonNull String value)` instead.
-
-#### SignUpRequest
-- `public SignUpRequest addAuthenticationParameters(@NonNull Map<String, Object> parameters)`. Use `public SignUpRequest addAuthenticationParameters(@NonNull Map<String, String> parameters)` instead.
-- `public SignUpRequest addSignUpParameters(@NonNull Map<String, Object> parameters)`. Use `public SignUpRequest addSignUpParameters(@NonNull Map<String, String> parameters)` instead.
-- `public SignUpRequest addParameters(@NonNull Map<String, Object> parameters)`. Use `public SignUpRequest addParameters(@NonNull Map<String, String> parameters)` instead.
-- `public SignUpRequest addParameter(@NonNull String name, @NonNull String value)`. Use `public SignUpRequest addParameter(@NonNull String name, @NonNull String value)` instead.
-- `public SignUpRequest setDevice(@NonNull String device)`. Use `public Request<Credentials, AuthenticationException> addParameter("device", value)` instead.
-
-#### AuthenticationRequest
-- `public AuthenticationRequest addAuthenticationParameters(@NonNull Map<String, String> parameters)`. Use `public Request<Credentials, AuthenticationException> addParameters(@NonNull Map<String, String> parameters)` instead.
-- `public AuthenticationRequest setDevice(@NonNull String device)`. Use `public Request<Credentials, AuthenticationException> addParameter("device", value)` instead.
+- `addAuthenticationParameters(parameters)` has been removed. Use `addParameters(parameters)` instead.
+- `setDevice("{DEVICE}")`. Use `addParameter("{DEVICE}", "{VALUE}")` instead.
 
-### Constants removed
+#### `WebAuthProvider.Builder` removals and changes
 
-- `ParameterBuilder.GRANT_TYPE_JWT` has been removed.
-- `ParameterBuilder.ID_TOKEN_KEY` has been removed.
-- `ParameterBuilder.DEVICE_KEY` has been removed.
-- `ParameterBuilder.ACCESS_TOKEN_KEY` has been removed.
-- `ResponseType.CODE`, `ResponseType.ID_TOKEN`, and `ResponseType.ACCESS_TOKEN` have been removed.
+- `useCodeGrant(boolean useCodeGrant)`. There is no replacement; only Code + PKCE flow supported in v2.
+- `useBrowser(boolean useBrowser)`. There is no replacement; Google no longer supports WebView authentication.
```

Comment:
Instead of Google, I'd use "this library". Same below

Reply:
Updated.